regolith/helpers/l_grantshelper.py

```diff
@@ -68,20 +66,39 @@ def sout(self):
         # print('rc.lead: {}'.format(rc.list))
```

Comment:
how about something like this. It is a bit more pythonic It would need is_current to be refactored:
```
for grant in gtx['grants']:
    grant_dates = get_dates(grant)
    if rc.current and not is_current(grant_dates['begin_date'], grant_dates['end_date']):
        continue
    print(f"{grant.get('alias', '')},  {grant.get('awardnr', '')}, "
             f"{str(grant.get('account', ""))}, {date['begin_date'].isoformat()}, {date['end_date'].isoformat())
```

Reply:
actually, on thinking about it, this is even nicer I think:
```
for grant in gtx['grants']:
    if rc.current and not is_current(grant):
        continue
    print(f"{grant.get('alias', '')},  {grant.get('awardnr', '')}, "
             f"{str(grant.get('account', ""))}, {date['begin_date'].isoformat()}, {date['end_date'].isoformat())
```
It could use the is_current I made for the tdd hackathon.  The best would be to replace the is_current() in regolith tools, but we would have to fix  everywhere it is used in regolith.  It should be quick though, and takes regolith in a good direction.  Could you take that on do you think?